@ecomplus/storefront-components/src/js/QuantitySelector.js

```diff
@@ -0,0 +1,66 @@
+import { i18n } from '@ecomplus/utils'
+import AAlert from '../AAlert.vue'
+
+export default {
+  name: 'QuantitySelector',
+
+  components: {
+    AAlert
+  },
+
+  data () {
+    return {
+      itemsSelected: [],
+      isMax: false
+    }
+  },
+
+  props: {
+    items: Array,
+    min: {
+      type: Number,
+      default: 1
+    },
+    max: {
+      type: Number,
+      default: 10
+    },
+    title: String
+  },
+
+  computed: {
+    selectedQuantity () {
+      return this.itemsSelected.map(item => parseInt(item.quantity, 10))
+    },
+    quantityRemaining () {
+      const total = this.selectedQuantity.reduce((a, b) => a + b, 0)
+      return this.max - total
```

Comment:
```suggestion
      return this.max
        ? this.max - this.selectedQuantity.reduce((a, b) => a + b, 0)
        : 9999999
```

Acho que `max` deve ser opcional...